packages/block-library/src/image/image.js

```diff
@@ -319,6 +333,23 @@ export default function Image( {
 						onChange={ updateAlignment }
 					/>
 				) }
+				{ ! isContentLocked && (
+					<ToolbarButton
+						onClick={ () => {
+							setShowCaption( ! showCaption );
+							if ( showCaption && ! caption ) {
+								setAttributes( { caption: undefined } );
+							}
+						} }
+						icon={ captionIcon }
+						isPressed={ showCaption }
+						label={
```

Comment:
@ntsekouras You can simply do this.

`label={ __( 'Caption' ) }`

No need to change the label since you added `aria-pressed`, that is good enough for communicating for A11Y purposes. It would save on additional verbosity.